bin/unistyle

```diff
@@ -35,14 +24,27 @@ var argv = yargs
   .alias('v', 'version')
   .argv;
 
-var compile = require('../src');
+var compile = require('../build');
 
-compile({file: argv._[0], output: argv.output}, function (err, css) {
-  if (err) {
-    console.error(err.stack);
-    return process.exit(1);
-  } else if (!argv.output) {
-    console.log(css);
+stdin().then(function (src) {
+  if (src) {
+    try {
+      src = requireFromString(src);
+    } catch (e) {
+      console.error(e);
+    }
+  } else {
+    src = require(resolve(argv._[0]));
   }
-  process.exit(0);
-});
+
+
+  compile({src: src, output: argv.output}, function (err, css) {
+    if (err) {
+      console.error(err.stack);
+      return process.exit(1);
```

Comment:
@sindresorhus yes, I know, but I think it's easier to grasp that the control flow is disrupted when using `return`, because one don't have to read what follows it, be it a callback or what not. Any counter opinion?
Will change this to a promisified API though :)